CONTRIBUTING.md

```diff
@@ -27,3 +42,38 @@ In order to accept your pull request, we need you to submit a CLA. You only need
 ## License
 
 By contributing to OCamlFormat, you agree that your contributions will be licensed under the LICENSE file in the root directory of this source tree.
+
+## Ocamlformat Labels
+
+### Pull Request specific
+
+- [CLA Signed](https://github.com/ocaml-ppx/ocamlformat/labels/CLA%20Signed): Added automatically as long as the user signed the Contributor License Agreement (see above)
+
+### Issue specific
+
+- [Good-first-issue](https://github.com/ocaml-ppx/ocamlformat/labels/Good-first-issue): This issue is a good entry point in the project for new contributors
+
+### Kind
+
+- [Kind/Bug](https://github.com/ocaml-ppx/ocamlformat/labels/Kind%2FBug): This issue describes a problem or this contribution addresses a problem (erroneous/unintended behavior)
+- [Kind/Docs](https://github.com/ocaml-ppx/ocamlformat/labels/Kind%2FDocs): This is related to a documentation change (user documentation or source code documentation)
+- [Kind/Feature-request](https://github.com/ocaml-ppx/ocamlformat/labels/Kind%2FFeature-request): This issue or contribution proposes a new feature
+- [Kind/Style-suggestion](https://github.com/ocaml-ppx/ocamlformat/labels/Kind%2FStyle%20suggestion): This issue or contribution proposes a style modification for the formatted output
+- [Kind/To-discuss](https://github.com/ocaml-ppx/ocamlformat/labels/Kind%2FTo-discuss): Discussion needed to converge to a solution
+
+### Priority
+
+- [Priority/Critical](https://github.com/ocaml-ppx/ocamlformat/labels/Priority%2FCritical): Highest priority, for blocking bugs or features that prevent ocamlformat to be used
+- [Priority/High](https://github.com/ocaml-ppx/ocamlformat/labels/Priority%2FHigh): For non-blocking bugs or features, that don't prevent ocamformat to be used
+- [Priority/Medium](https://github.com/ocaml-ppx/ocamlformat/labels/Priority%2FMedium): For important documentation and style suggestions
+- [Priority/Low](https://github.com/ocaml-ppx/ocamlformat/labels/Priority%2FLow): Lowest priority, for less important documentation and style suggestions
+
+### Status
+
+- [Status/0-More-info-needed](https://github.com/ocaml-ppx/ocamlformat/labels/Status%2F0-More-info-needed): More information is needed before this issue can be triaged
+- [Status/0-Triage](https://github.com/ocaml-ppx/ocamlformat/labels/Status%2F0-Triage): This issue needs triaging
+- [Status/1-Acknowledged](https://github.com/ocaml-ppx/ocamlformat/labels/Status%2F1-Acknowledged): This issue has been triaged and is being investigated
+- [Status/2-Regression](https://github.com/ocaml-ppx/ocamlformat/labels/Status%2F2-Regression): Known workaround to be applied and tested
```

Comment:
This one is surprising to me. I assumed that it meant an issue present in the current version but not in an earlier one, that is, an issue introduced by a change, rather than something that never worked.